src/drivers/driver_postgres.rs

```diff
@@ -0,0 +1,25 @@
+#![cfg(feature = "driver-postgres")]
+use drivers::Driver as Driv;
+
+pub struct Driver{
+    target: &'static str,
+}
+
+impl Driv for Driver {
```

Comment:
`DriverTrait` would be better. Personally, I prefer verbosity whenever possible. If I have to choose between verbose variable names and comments everywhere, I'd choose verbose variables.